MapboxCoreNavigation/RouteController.swift

```diff
@@ -139,6 +142,11 @@ open class RouteController: NSObject {
         super.init()
         
         self.locationManager.delegate = self
+        
+        // This is just a test!!! (needs a real access token, a user agent base, and the actual nav SDK version
+        events.initialize(withAccessToken: "your token here!", userAgentBase: "MapboxEventsiOSNavigation", hostSDKVersion: "42.42.42")
+        events.isDebugLoggingEnabled = true
+        events.sendTurnstileEvent()
```

Comment:
@bsudekum is correct, this was just an example. However, as discussed, we are implementing a universal turnstile event that all host SDKs will use.